src/shipit_risk_assessment/VERSION

```diff
@@ -1 +1 @@
-1.0.0
+0.1.0
```

Comment:
Good point. As no version of this project has reached production, i thought we could start from 0.1.0